SofascoreAcademy/SofascoreAcademy/View/ViewModel/TeamScoreViewModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  TeamScoreViewModel.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 21.03.2024..
+//
+
+import UIKit
+
+struct TeamScoreViewModel {
+    
+    let teamName: String
+    let logoImage: UIImage
+    let teamScore: String
+    
+    let teamNameColor: UIColor
+    let teamScoreColor: UIColor
+    
+    init(teamModel: Team, teamScore: String, scoreStatus: ScoreStatus) {
+        self.teamName = teamModel.teamName
+        self.logoImage = teamModel.logoImage
+        self.teamScore = teamScore
+        
+        switch scoreStatus {
+        case .notFinished:
+            teamNameColor = .onSurfaceLv1
+            teamScoreColor = .specificLive
+        case .won:
+            teamNameColor = .onSurfaceLv1
+            teamScoreColor = .onSurfaceLv1
+        case .lost, .draw:
+            teamNameColor = .onSurfaceLv2
+            teamScoreColor = .onSurfaceLv2
+        }
```

Comment:
enumi su super, jer možeš unutar imati definirane var i func za to okruženje, i unutar tih varijabli raditi switch. Na taj način, ako ti treba negdje vani neki specifični switch imaš ga na jednom mjestu.

U ovom slučaju ako to napraviš, imati ćeš jednostavniji view. Evo ti hint. 

Tako da unutar enum ScoreStatus, možeš imati 
var teamNameColor: UIColor {
switch self {
}
}
...